data/json/mutations/mutations.json

```
@@ -831,7 +831,7 @@
     "id": "XS",
     "name": { "str": "Weight: XS" },
     "points": -1,
-    "description": "You lost an awful lot of weight before Cataclysm.  Whatever the reason, your skeletal figure is now an extra burden, when food is not at hand's reach.  You need to gain weight or die trying.",
+    "description": "You lost an awful lot of weight before the Cataclysm.  Whatever the reason, now that food is not always close at hand, your skeletal figure is now an additional burden.  You need to gain weight or die trying.",
```

```suggestion
    "description": "You lost an awful lot of weight before the Cataclysm.  Whatever the reason, now that food is not always close at hand your skeletal figure is an additional risk.  You need to gain weight or die trying.",
```
"burden" is not quite right for reduced weight.

---

Ehh. The condition of being underweight is a burden to survival is how I read it.

---

That is certainly true; it's just that there are alternative words (like risk, as in risk of starving) that IMO sound better.

---

I'll swap in "danger" for both traits.